src/locales/en/common.json

```diff
@@ -361,7 +362,6 @@
   "Number of transactions / Volume (LSK)": "Number of transactions / Volume (LSK)",
   "Official guidelines": "Official guidelines",
   "Ok": "Ok",
-  "Old account": "Old account",
```

Comment:
like wise this, it seems weird this text was removed by i18n even though there are no changes to the JSX in this PR

Reply:
its not exist in project